EasyPost/Models/API/Payload.cs

```diff
@@ -0,0 +1,13 @@
+namespace EasyPost.Models.API
+{
+    public class Payload : Event // A payload has the same schema as an Event
```

Comment:
I am not sure if this is true, according to the Go: https://github.com/EasyPost/easypost-go/blob/master/event.go#L35-L51

Reply:
Oh yeah, this is definitely not true.